src/models/spaces/organisms/organisms-row.ts

```diff
@@ -30,36 +30,52 @@ export const OrganismsRowModel = types
     hoveredOrganelle: types.maybe(Organelle),
     selectedOrganelle: types.maybe(Organelle),
     mode: types.optional(Mode, "normal"),
-    assayedOrganelle: types.maybe(Organelle),
+    assayedOrganelles: types.array(Organelle),
     zoomLevel: types.optional(ZoomLevel, "organism"),
     showProteinDNA: false,
     showProteinAminoAcidsOnProtein: false,
     proteinSliderStartPercent: 0
   })
   .views(self => ({
     get currentData(): ChartDataModelType {
-      const chartDataSets = [];
-      const points: DataPointType[] = [];
-      const organelle = self.assayedOrganelle;
-      if (organelle) {
-        kSubstanceNames.forEach((substance) => {
+      const chartDataSets: any[] = [];
```

Comment:
I think this could be type `ChartDataSetModelType[]`

Reply:
Yup.  Will update.